test/uk/gov/hmrc/helptosave/modules/UCThresholdOrchestratorSpec.scala

```diff
@@ -62,7 +62,7 @@ class UCThresholdOrchestratorSpec extends ActorTestSupport("UCThresholdOrchestra
 
       connector
         .getThreshold()(*, *)
-        .returns(Future.successful(HttpResponse(500, "")))
+        .returns(Future.successful(Right(HttpResponse(500, ""))))
```

Comment:
This doesn't seem rite. With the new error handling in HttpClientV2, we will never get a right based HttpResponse for 4xx and 5xx. I would check that the connector function has correct return type and that we handle this as a Left on the error string or whatever is relevant to check here.